src/main/java/com/example/examplemod/ExampleMod.java

```diff
@@ -4,6 +4,8 @@
 import cpw.mods.fml.common.Mod;
 import cpw.mods.fml.common.Mod.EventHandler;
 import cpw.mods.fml.common.event.FMLInitializationEvent;
+import cpw.mods.fml.common.FMLLog;
+import java.lang.Byte;;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid importing anything from the package java.lang](https://www.codacy.com/app/shardrill_t/playground/file/2177672016/issues/source?bid=3163820&fileBranchId=3163861#l8)